packages/account-management/src/frontend/Registration/Registration.tsx

```diff
@@ -56,6 +56,9 @@ export const Registration: React.FC = () => {
     setValid(!!(firstName && lastName && email && passwordPolicy.test(pass || '')));
   }, [firstName, lastName, email, pass, consent]);
 
+  const isFirstNameValid = Boolean(firstName && firstName !== '');
+  const isLastNameValid = Boolean(lastName && lastName !== '');
```

Comment:
Following the conversation on Friday, we don't actually need `Boolean` or `!!` here!

The right side of the `&&` evaluates to a boolean already. So we either get `false` because the name isn't set, or `false` because it's an empty string, or `true` when it's not empty.

Reply:
@ajrussellaudio taking out `Boolean()` or `!!` here results in the following error on the type check: 

```
Type 'boolean | "" | undefined' is not assignable to type 'boolean | undefined'.
  Type '""' is not assignable to type 'boolean | undefined'.
```

Reply:
Yikes. How about:

```ts
const isFirstNameValid = firstName && (firstName !== '');
```

Reply:
Yup, this results in the same type error!

Reply:
Dammit, TypeScript! 😅 

If that's the case, your solution is the right one 👍